index.js

```diff
@@ -32,8 +33,22 @@ const asyncTask = async (title, task) => {
 	stop('Success');
 };
 
+await asyncTask('Get Latest tag', async ({ log }) => {
+	const listText = await git.listRemote(["--tags", "--sort=-v:refname", SHEETJS_GIT_REPOSITORY_URL]);
+	const listTags = listText.match(/[^\r\n]+/g)
+		.map(s => s.replace(/^.*refs\/tags\//, ''))
+		.filter(t => semver.valid(t.substring(1)))			// check valid version
+		.filter(t => /^v[0-9]+\.[0-9]+\.[0-9]+$/.test(t))	// skip -a, -h, -i, +deno  etc
+	latestTagName = listTags[0];
```

Comment:
It seems [simple-git](https://www.npmjs.com/package/simple-git) has a `.tags()` method that allows you to return what you need. Have you tried using it?

![image](https://github.com/e965/sheetjs-npm-publisher/assets/18071040/799dc7f5-c167-45a2-a073-05af61e9ac52)

Reply:
If we improve on the current state of using .listRemote(), it can no longer receive git clone if update is not needed.

However, .tags() must be used after git clone, and then checkout with the confirmed latest tag.

Reply:
I will change it to clone only when there is a new version.